Notebooks/Metrics.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>It would be nicer to have the coefficients in the get_distance_* functions to be a variable.</p><p>for example:</p><p>--</p><p><code>def get_distance_road(lat_a, long_a, lat_b, long_b):</code></p><p><code>	henning_coefficient = 1.3</code></p><p><code>	return geodesic((lat_a, long_a), (lat_b, long_b)).km * henning_coefficient</code></p><p>This is easier to understand why there is a hard coded value and it will make it a bit easier to adjust in the future or even call another function if we get better input.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AndOleAnd/Capstone_N_A_P/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>